web/src/styles/_btns.scss

```diff
@@ -1,3 +1,3 @@
 .btn-primary {
-	@apply border-solid border-[1px] rounded-xl border-secondary text-secondary py-3 px-6 transition-all hover:bg-secondary hover:text-white;
+	@apply border-solid border-[1px] rounded-[35px] border-primary font-normal text-primary text-[24px] py-[16px] px-[48px] transition-colors hover:bg-primary-opacity;
```

Comment:
same here with the border radius and the padding

Reply:
done